src/p5.dance.js

```diff
@@ -184,6 +184,11 @@ module.exports = class DanceParty {
   }
 
   async loadCostumeAnimations(costume, costumeData) {
+    if (!this.animations[costume]) {
+      // Invalid costume, nothing to do:
```

Comment:
Add a console statement similar to line 974.

Reply:
Done.